lockup/src/getters.rs

```diff
@@ -61,12 +61,19 @@ impl LockupContract {
 
     /// Get the amount of tokens that are locked in this account due to lockup or vesting.
     pub fn get_locked_amount(&self) -> WrappedBalance {
-        if let LockupStartInformation::TransfersEnabled { lockup_timestamp } =
-            &self.lockup_information.lockup_start_information
+        if let TransfersInformation::TransfersEnabled {
+            transfers_timestamp,
+        } = &self.lockup_information.transfers_information
         {
-            let lockup_timestamp = lockup_timestamp
-                .0
-                .saturating_add(self.lockup_information.lockup_duration.0);
+            let lockup_timestamp = std::cmp::max(
```

Comment:
why is it a maximum of the two here? shouldn't it always be transfer_timestamp + lock duration?

Reply:
We lock tokens to whichever is later. E.g. `40` days lock from the date of X or `14` days after the vote.